tests/common/fixtures/ptfhost_utils.py

```diff
@@ -269,8 +269,17 @@ def run_icmp_responder_session(duthosts, duthost, ptfhost, tbinfo):
 
     yield
 
-    # NOTE: Leave icmp_responder running for dualtor-mixed topology
-    return
+    if "dualtor-mixed" in tbinfo["topo"]["name"] or "dualtor-aa" in tbinfo["topo"]["name"]:
```

Comment:
should we leave `icmp_responder` running here?

Reply:
> should we leave `icmp_responder` running here?

Noticed `icmp_responder` was always stopped after tests on `dualtor` topo testbeds. I don't know the historical reason behind this, so I just stop `icmp_responder` at the end of the test session,

Reply:
Can we enforce the same behavior for all dualtor testbeds(`dualtor`, `dualtor-mixed` or `dualtor-aa`)? Leave it running after each session could reduce any inconsistencies between sessions.

Reply:
Sure.